_includes/api/en/5x/menu.md

```diff
@@ -4,8 +4,10 @@
     <ul id="express-menu">
         <li><em>Methods</em></li>
         <li id="express-json-middleware"><a href="#express.json">express.json()</a></li>
-        <li id="express-static-middleware"><a href="#express.static">express.static()</a></li>
+        <li id="express-raw-middleware"><a href="#express.raw">express.raw()</a></li>
         <li id="express-router"><a href="#express.router">express.Router()</a></li>
+        <li id="express-static-middleware"><a href="#express.static">express.static()</a></li>
+        <li id="express-text-middleware"><a href="#express.text">express.text()</a></li>
```

Comment:
I've added the menu entries with `id`s suffixed with `-middleware` for consistency, but in 4.x `express.raw()` and `express.text()` menu items have no such suffix in `id`s.

I don't think anyone makes links with anchors to menu elements, so it should be ok.

https://github.com/expressjs/expressjs.com/blob/9e00f5aa328bb3d43d8d3f3f4a5d22bf6d5ef19c/_includes/api/en/4x/menu.md?plain=1#L6-L11